stdlib.go

```diff
@@ -52,6 +54,11 @@ func MessageKey(key string) StdlibAdapterOption {
 	return func(a *StdlibAdapter) { a.messageKey = key }
 }
 
+// WithCaller defines, if StdlibAdapter should expect a caller in the message.
+func WithCaller(expectCaller bool) StdlibAdapterOption {
```

Comment:
I'm not convinced `WithCaller` is the best name here. In my experience `With...` functions add data or behavior to something. Although that's sort of what's going on here, I think it's more accurate to think of this as controlling what the StdLibAdapter should expect in the data it's processing, which your parameter name does a better job conveying. Staying consistent with the `FileKey` option above that is related the same log data perhaps `ExpectFile` works better here.

What do you think of this?
```
func ExpectFile(expected bool) StdlibAdapterOption
```

Also, I think the word "defines" in the comment is a bit out of place and the comment should provide more details. Since the `Prefix` option below allows a similar type of control as the new option in this PR consider taking the comment from Prefix as a template and adjusting it to fit this use case. I would also add some explanation for why someone would want to use set this option to `false` since the default `true` doesn't _require_ log messages to contain caller information.

Reply:
I use Regexp as function name now. otherwise I could use `LogRegexp` as function name too. What would you prefer?